pymc3/distributions/distribution.py

```diff
@@ -403,6 +433,8 @@ def __init__(
             If ``True``, the shape of the random samples generate in the
             ``random`` method is checked with the expected return shape. This
             test is only performed if ``wrap_random_with_dist_shape is False``.
+        givens : dict, optional
+            Model variables on which the DensityDist is conditioned.
```

Comment:
why not include this in the arguments to `__init__`?

Reply:
I _think_ that `givens` (or `given` if it's renamed) will never get to be passed to `__init__` because it's popped from kwargs in `__new__`, but truth be told, even after getting this to work I am not yet completely sure I understand new vs init. I have added it to the docstring anyways because of its special use tied to DensityDist and MultiObservedRV to make sure it's documented somewhere people searching for it can find it.